OpenRA.Mods.Common/Traits/Render/DrawLineToTarget.cs

```diff
@@ -37,16 +38,14 @@ public DrawLineToTarget(Actor self, DrawLineToTargetInfo info)
 
 		public void SetTarget(Actor self, Target target, Color c, bool display)
 		{
-			targets = new List<Target> { target };
 			this.c = c;
 
 			if (display)
 				lifetime = info.Delay;
 		}
 
-		public void SetTargets(Actor self, List<Target> targets, Color c, bool display)
+		public void SetTargets(Actor self, Color c, bool display)
```

Comment:
It would be much more robust to remove this completely, and have `GetTargets` return the color as well as the destination.